ethereum/spec/src/testFixtures/java/tech/pegasys/teku/spec/util/BeaconStateBuilder.java

```diff
@@ -109,54 +105,63 @@ private void initDefaults() {
     latestBlockHeader = dataStructureUtil.randomBeaconBlockHeader();
     blockRoots =
         dataStructureUtil.randomSSZVector(
-            Bytes32.ZERO, Constants.SLOTS_PER_HISTORICAL_ROOT, dataStructureUtil::randomBytes32);
+            Bytes32.ZERO,
+            dataStructureUtil.getSlotsPerHistoricalRoot(),
```

Comment:
oo, interesting approach. Useful I guess but I'm a bit worried about `DataStructureUtil` becoming the proxy for `SpecConstants`.  Once everything is converted over would be reasonable for `DataStructureUtil` to have a `getSpecConstants` (which by then wouldn't be optional) and then could inline this.

So yeah, seems like a good approach to ease the transition.

Reply:
and specifically this is basically owned by DataStructureUtil i guess is my thinking...